package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "fh-reportingclient",
   "description": "FH Reporting Client",
-  "version": "0.5.7",
+  "version": "0.5.8",
```

Comment:
Do you plan on releasing this version once merged? Are there other any other changes to be released apart from the `engines.node` update?

Reply:
Since here we have the following changes:

* Remove the deprecated turbo dependence and use mocha instead of it.
* Upgrade nodejs engine
* Upgrade dev dependencies only.

I think we should bump the version.

Reply:
The replacement of turbo with mocha, and upgrading of dev dependencies, would not warrant a version bump since they don't change the program in any way, so there's nothing to be released. You could argue that upgrading the minimum nodejs version is worthy of a release or not, and also the huge changes to the shrinkwrap file.

I guess you just have to decide if the changes to the nodejs engine and the shrinkwrap file are valuable enough to users of this module to warrant a release by themselves.

Reply:
Hi @grdryn, 

Following the points. 

1) I came back the original shrinkwrap, however, I think we should regenerate it. Following the reason.  ( note that the change is not in the versions used but in the structure of the file )

```
pm WARN read-shrinkwrap This version of npm is compatible with lockfileVersion@1, but npm-shrinkwrap.json was generated for lockfileVersion@0. I'll try to do my best with it!
npm WARN deprecated is-my-json-valid@2.16.0: catastrophic backtracking in regexes could potentially lead to REDOS attack, upgrade to 2.17.2 as soon as possible
````
What do you think?

2) Regards, bump the version: IHMO we should do that, however, if you think that it can cause any impact we can add it in a CHANGELOG as unreleased change. What do you think?

Reply:
@camilamacedo86 I will leave that to @davidffrench and yourself to decide what to do here. :+1:

Reply:
I am not worried about the npm warning for the shrinkwrap. I agree with @grdryn , If we are only updating devDependencies, there is no need to update the shrinkwrap. 

Even though the engines change is small, I believe we should release a new version to npm so a version bump would be needed. @camilamacedo86 Please make sure to publish to npm once this is merged.